api/src/pcapi/utils/requests.py

```diff
@@ -27,25 +28,33 @@
 REQUEST_TIMEOUT_IN_SECOND = 10
 
 
+def _react_url(url: str) -> str:
+    # Cine Digital Service (CDS) wants authentication token to appear in GET
+    # requests. We don't want to log them.
+    return re.sub("api_token=[^&^$]+", "api_token=[REDACTED]", url)
+
+
 def _wrapper(request_func: Callable, method: str, url: str, log_at_error_level=True, **kwargs: Any) -> Response:  # type: ignore [no-untyped-def]
     timeout = kwargs.pop("timeout", REQUEST_TIMEOUT_IN_SECOND)
     try:
         response = request_func(method=method, url=url, timeout=timeout, **kwargs)
     except Exception as exc:
+        redacted_url = _react_url(url)
         # If the URL is not controlled by us, there is probably no
         # reason to send the error to Sentry. Logging an INFO message
         # is enough if we ever need to debug.
         if log_at_error_level:
             logger_method = logger.exception
         else:
             logger_method = logger.info
-        logger_method("Call to external service failed with %s", exc, extra={"method": method, "url": url})
+        logger_method("Call to external service failed with %s", exc, extra={"method": method, "url": redacted_url})
```

Comment:
je pense que tu peux appeler directement la fonction ici : 

```suggestion
        logger_method("Call to external service failed with %s", exc, extra={"method": method, "url": _redact_url(url)})
```

idem plus bas